internal/util/init_context.go

```diff
@@ -19,28 +20,33 @@ func InitUDMContext(udmContext *context.UDMContext) {
 	sbi := configuration.Sbi
 	udmContext.UriScheme = ""
 	udmContext.SBIPort = factory.UdmSbiDefaultPort
-	udmContext.RegisterIPv4 = factory.UdmSbiDefaultIPv4
+	udmContext.RegisterIP = factory.UdmSbiDefaultIP
 	if sbi != nil {
```

Comment:
Also ra andom question, what's the difference between this InitUDMContext and the one in internal/context/context.go? When one is used and the other one is used? Thanks :-)

Reply:
There is not, it's was introduced by the first commit and never use by the function Start
```
// udm_util.InitUDMContext(self)
udm_context.Init()
```
It's a dead code